transmart-core-db/grails-app/services/org/transmartproject/db/clinical/PatientSetService.groovy

```diff
@@ -439,7 +439,13 @@ class PatientSetService extends AbstractDataResourceService implements PatientSe
         return PersistSubqueriesResult.notPatientSets(constraint)
     }
 
-    private Criterion buildSubselectCriterion(String property, Constraint constraint) {
+    /**
+     *
```

Comment:
it looks like it was planned to add a description here

Reply:
yeah, I've looked in the method. I could not come up with a sensible description. IMHO the method does to many things at once.